Cesium.CodeGen/Ir/Declarations/LocalDeclarationInfo.cs

```diff
@@ -303,6 +303,7 @@ private static (IType, string? Identifier) ProcessDirectDeclarator(IDirectDeclar
                 ("long", null, null, null) => PrimitiveTypeKind.Long,
                 ("float", null, null, null) => PrimitiveTypeKind.Float,
                 ("double", null, null, null) => PrimitiveTypeKind.Double,
+                ("nint", null, null, null) => PrimitiveTypeKind.NativeInt,
```

Comment:
What would you say about calling that either `__nint` or `_Nint`? more in line with how the standard works.

(and then, we _could_ introduce a Cesium-only file that would add a `typedef nint __nint`, if really required)

Reply:
I dont belive we need both `__nint` and `nint`. I only want expose that implementation detail, so we can model `ptrdiff_t`, `intptr_t` and `uintptr_t` properly. I do not even want to document that type.